### ISSUE TITLE
Android build fail: Could not resolve com.android.tools.build:gradle:3.1.4

### ISSUE THREAD

**Issue by retyui** [segmentio/analytics-react-native#525]
- `analytics-react-native` version: 1.5.0
- Integrations versions (if used): ???
- React Native version: 0.66.x
- Android?

**Steps to reproduce**

1. Add this module 1.5.0 to project 
2. Try to build Android

**Expected behavior**

Android build won't fail

**Actual behavior**

```tsx

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':segment_analytics-react-native'.
> Could not resolve all artifacts for configuration ':segment_analytics-react-native:classpath'.
   > Could not resolve com.android.tools.build:gradle:3.1.4.
     Required by:
         project :segment_analytics-react-native
      > Could not resolve com.android.tools.build:gradle:3.1.4.
         > Could not get resource 'https://jcenter.bintray.com/com/android/tools/build/gradle/3.1.4/gradle-3.1.4.pom'.
            > Could not GET 'https://jcenter.bintray.com/com/android/tools/build/gradle/3.1.4/gradle-3.1.4.pom'.
               > Read timed out
   > Could not resolve org.jetbrains.kotlin:kotlin-compiler-embeddable:1.4.0.
     Required by:
         project :segment_analytics-react-native > org.jetbrains.kotlin:kotlin-gradle-plugin:1.4.0
      > Skipped due to earlier error
   > Could not resolve org.jetbrains.kotlin:kotlin-annotation-processing-gradle:1.4.0.
     Required by:
         project :segment_analytics-react-native > org.jetbrains.kotlin:kotlin-gradle-plugin:1.4.0
      > Skipped due to earlier error
   > Could not resolve org.jetbrains.kotlin:kotlin-android-extensions:1.4.0.
     Required by:
         project :segment_analytics-react-native > org.jetbrains.kotlin:kotlin-gradle-plugin:1.4.0
      > Skipped due to earlier error
   > Could not resolve org.jetbrains.kotlin:kotlin-compiler-runner:1.4.0.
     Required by:
         project :segment_analytics-react-native > org.jetbrains.kotlin:kotlin-gradle-plugin:1.4.0
      > Skipped due to earlier error
   > Could not resolve org.jetbrains.kotlin:kotlin-scripting-compiler-embeddable:1.4.0.
     Required by:
         project :segment_analytics-react-native > org.jetbrains.kotlin:kotlin-gradle-plugin:1.4.0
      > Skipped due to earlier error
   > Could not resolve org.jetbrains.kotlin:kotlin-scripting-compiler-impl-embeddable:1.4.0.
     Required by:
         project :segment_analytics-react-native > org.jetbrains.kotlin:kotlin-gradle-plugin:1.4.0
      > Skipped due to earlier error
   > Could not resolve org.jetbrains.kotlin:kotlin-reflect:1.4.0.
     Required by:
         project :segment_analytics-react-native > org.jetbrains.kotlin:kotlin-gradle-plugin:1.4.0
      > Skipped due to earlier error
   > Could not resolve org.jetbrains.kotlin:kotlin-gradle-plugin-api:1.4.0.
     Required by:
         project :segment_analytics-react-native > org.jetbrains.kotlin:kotlin-gradle-plugin:1.4.0
      > Skipped due to earlier error
   > Could not resolve org.jetbrains.kotlin:kotlin-gradle-plugin-model:1.4.0.
     Required by:
         project :segment_analytics-react-native > org.jetbrains.kotlin:kotlin-gradle-plugin:1.4.0
      > Skipped due to earlier error
   > Could not resolve org.jetbrains.kotlin:kotlin-stdlib:1.4.0.
     Required by:
         project :segment_analytics-react-native > org.jetbrains.kotlin:kotlin-gradle-plugin:1.4.0
      > Skipped due to earlier error
   > Could not resolve org.jetbrains.kotlin:kotlin-util-klib:1.4.0.
     Required by:
         project :segment_analytics-react-native > org.jetbrains.kotlin:kotlin-gradle-plugin:1.4.0
      > Skipped due to earlier error
   > Could not resolve de.undercouch:gradle-download-task:4.0.2.
     Required by:
         project :segment_analytics-react-native > org.jetbrains.kotlin:kotlin-gradle-plugin:1.4.0
      > Skipped due to earlier error
   > Could not resolve com.github.gundy:semver4j:0.16.4.
     Required by:
         project :segment_analytics-react-native > org.jetbrains.kotlin:kotlin-gradle-plugin:1.4.0
      > Skipped due to earlier error

```

**Comment by VinceAbruzzese** [comment]
Failing for me as well, looks like jcenter is returning a 403 error. https://jcenter.bintray.com/

**Comment by alanjcharles** [comment]
Hello @retyui  @VinceAbruzzese, please update to the latest version as it no longer relies on `jcenter`. You can find the [migration guide here](https://github.com/segmentio/analytics-react-native/blob/master/MIGRATION_GUIDE.md). Be sure to add `@segment/sovran-react-native` as well as `@react-native-async-storage/async-storage`. Please feel free to reach out to friends@segment.com for additional support.

**Comment by retyui** [comment]
@alanjcharles I will happy to update a segment module to `2.x` but it no possible right now 

I need just unlock our release process.

Currently `1.5.x` the most popular version and I think it make sense to make a small hotfix for it:

![image](https://user-images.githubusercontent.com/95620376/166744290-5928dcbd-22c4-4360-b92a-9ab051d25e2c.png)

See: https://www.npmjs.com/package/@segment/analytics-react-native

**Comment by oscb** [comment]
@retyu I just released [v1.5.3](https://www.npmjs.com/package/@segment/analytics-react-native/v/1.5.3) which replaces jcenter for maven in all packages.

Let me know if this unblocks you.

**Comment by retyui** [comment]
@oscb Confirm, now it works
